arangod/ClusterEngine/ClusterTransactionState.cpp

```diff
@@ -80,7 +80,7 @@ Result ClusterTransactionState::beginTransaction(transaction::Hints hints) {
   updateStatus(transaction::Status::RUNNING);
   _vocbase.server().getFeature<MetricsFeature>().serverStatistics()._transactionsStatistics._transactionsStarted++;
 
-  transaction::ManagerFeature::manager()->registerTransaction(id(), isReadOnlyTransaction());
+  transaction::ManagerFeature::manager()->registerTransaction(id(), isReadOnlyTransaction(), hasHint(transaction::Hints::Hint::IS_FOLLOWER_TRX));
```

Comment:
why would this be supported on the coordinator

Reply:
Just for uniformity. On a coordinator no transaction will ever be a "followertransaction". We might also just put `false` here, if you prefer.

Reply:
I also think we should hard-code this to `false`.

Reply:
Agreed, did the change. But I think this is only necessary for `devel`, not for the already merged PRs in the other branches.